src/Compilers/CSharp/Portable/FlowAnalysis/AbstractRegionDataFlowPass.cs

```diff
@@ -44,10 +50,39 @@ public override BoundNode VisitLambda(BoundLambda node)
 
         public override BoundNode VisitLocalFunctionStatement(BoundLocalFunctionStatement node)
         {
+            _badRegion = true;
             MakeSlots(node.Symbol.Parameters);
             return base.VisitLocalFunctionStatement(node);
         }
 
+        public override BoundNode VisitCall(BoundCall node)
+        {
+            if (node.Method.MethodKind == MethodKind.LocalFunction)
+            {
+                _badRegion = true;
```

Comment:
>  _badRegion = true; [](start = 14, length = 20)

Shouldn't this be done conditionally based on whether we are inside the region? Do we have tests for the API when local function is declared, but is referenced only outside the region?